docs/README.md

```diff
@@ -88,6 +88,18 @@ For example:
 },
 ```
 
+Example using some variables to render unit multiplier and measurement unit only on products that have it:
```

Comment:
I think we should make it clear that it only works for `product-selling-price`

Reply:
Good idea!